HyperPlay-Gaming/hyperplay-desktop-client
# [Tech] Pull Heroic 2.9.2 changes

Pull most of the Heroic changes of the latest release.
Full changelog: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/tag/v2.9.2

## How to Test

Most of the changes are related to the other store's backend and the compatibility layer. Should not break anything related to HyperPlay games but it is possible.
This is not a lot of changes as well like the previous PR that got 3 full Heroic releases.

1. Check main features of Epic and Gog games (Login, List Games, GamePage, install, uninstall, play, etc)
2. Check main HyperPlay games features as well.
3. Test Windows games on macOS and Linux using Wine or Proton.
4. Games that were working before should continue working. 

---

Use the following Checklist if you have changed something on the Backend or Frontend:
